src/base/bittorrent/session.cpp

```diff
@@ -2431,17 +2424,25 @@ QStringList Session::getListeningIPs() const
     const QString ifaceName = networkInterface();
     const QString ifaceAddr = networkInterfaceAddress();
     const QHostAddress configuredAddr(ifaceAddr);
-    const bool allIPv4 = (ifaceAddr == QLatin1Char('4')); // Means All IPv4 addresses
-    const bool allIPv6 = (ifaceAddr == QLatin1Char('6')); // Means All IPv6 addresses
+    const bool allIPv4 = (ifaceAddr == QStringLiteral("0.0.0.0")); // Means All IPv4 addresses
+    const bool allIPv6 = (ifaceAddr == QStringLiteral("::")); // Means All IPv6 addresses
```

Comment:
AFAIK you are duplicating the same string multiple times, i.e. every QStringLiteral() you constructed is going to be distinct in the binary and thus boating it up. IMO using QLatin1String() is suffice.